playwright/playwright.config.ts

```diff
@@ -51,7 +51,7 @@ const config: PlaywrightTestConfig = {
     /* Base URL to use in actions like `await page.goto('/')`. */
     baseURL: 'http://localhost:2900',
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
-    trace: process.env.CI ? 'retain-on-failure' : 'on',
+    trace: process.env.CI ? 'on-first-retry' : 'on',
```

Comment:
This should speed up playwright a lot because we're not doing traces on every test